cli/bin/test-binary.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+BASEDIR=`dirname $0`/..
+
+cd $BASEDIR
+
+PATH=$BASEDIR/../dist:$PATH
+source env/bin/activate
+cp tests/data/dcos.toml $DCOS_CONFIG
+py.test tests/integrations
```

Comment:
I think you can avoid the activate and deactivate by doing `env/bin/py.test tests/integrations`.